### PR TITLE
GH-1104: Factor build code out of WFL.

### PR DESCRIPTION
**Pull request by tbl3rd**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1104

### Changes

- Run System/exit to avoid waiting for the JVM thread scheduler to time out.

### Review Instructions

- The build functions run longer than it seems they should now. Not sure why yet.

### CODE REVIEW

File: api/module.mk [hunk 1]
```diff
@@ -33,14 +33,14 @@ JAR_LINK     := $(DERIVED_TARGET_DIR)/wfl.jar
 
 $(PREBUILD): $(SCM_SRC) $(SCM_RESOURCES)
```
Comment (ehigham) [inline api/module.mk:34]: I think you can remove the dependence on SCM_SRC here as the only thing that affects prebuild is your new build.clj file + this + deps.edn?

**Review by ehigham — APPROVED** [review]
noice!

File: api/module.mk (hunk 1 above)
Comment (tbl3rd) [reply api/module.mk]: Yes. Thanks.

**Review by rexwangcc — APPROVED** [review]
Looks great!

**Pull request merged**